Restore.cmd

```diff
@@ -55,6 +55,7 @@ echo Restoring packages: Templates
 call "%NugetExe%" restore "%RoslynRoot%src\Setup\Templates\Templates.sln" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
 
 echo Restoring packages: Roslyn (this may take some time)
+call "%NugetExe%" restore "%RoslynRoot%build\Toolset.sln" %NuGetAdditionalCommandLineArgs% || goto :RestoreFailed
```

Comment:
This would imply there are projects in Toolset.sln that aren't in Roslyn.sln. Is that the case?